game_api.txt

```diff
@@ -155,7 +155,7 @@ The doors mod allows modders to register custom doors and trapdoors.
 ### Fence gate definition
 
 	description = "Wooden Fence Gate",
-	texture = "default_wood.png",
+	texture = "default_wood.png", -- `backface_culling` will automatically be set to `true` if it's not given.
```

Comment:
Comments are part of code, and are part of code style guidelines, both MT and linux kernel guidelines. The column limit applies to comments too because when you split your screen left/right you need to be able to read comments, they are just as important to see as the code.